src/scripts/components/dialog/text-box.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react/addons';
+/**
+ *
+ */
+const SingleBox = React.createClass({
+	propTypes: {
+		content: React.PropTypes.string,
+		class:  React.PropTypes.string
```

Comment:
I think `class` is a reserved word. Use something else instead, like `className`.